project-koku/koku
# Initial notification stubs

Fixes [COST-2718](https://issues.redhat.com/browse/COST-2718)

Changes proposed in this PR:
* Adding notifications integration
* Adding first notification for OCP source missing cost model

Testing Instructions:
1. Add OCP sources with and without a cost model assigned
2. Jump in a django shell `python manage.py shell`
3. Run the following

from masu.celery.tasks import check_cost_model_status
check_cost_model_status()

You should see something like `Cost model status check finished. 1 notifications fired and 1 skipped`
+ a bunch of kafka errors :sweat_smile: Unless you spin up a kafka instance locally too
